tsconfig.json

```diff
@@ -9,40 +9,18 @@
     "declaration": true,
     "noImplicitAny": true,
     "removeComments": false,
-    "noLib": true,
     "preserveConstEnums": true,
     "suppressImplicitAnyIndexErrors": false,
     "outDir": "lib",
     "rootDir": "src"
   },
-  "filesGlob": [
-    "node_modules/typescript/lib/lib.core.es7.d.ts",
-    "custom_typings/*.d.ts",
-    "src/**/*.ts",
-    "typings/index.d.ts"
-  ],
   "files": [
-    "node_modules/typescript/lib/lib.core.es7.d.ts",
-    "custom_typings/dom5.d.ts",
-    "custom_typings/fs-extra.d.ts",
-    "custom_typings/hydrolysis.d.ts",
-    "custom_typings/node.d.ts",
-    "custom_typings/plylog.d.ts",
-    "custom_typings/slash.d.ts",
-    "custom_typings/sw-precache.d.ts",
-    "custom_typings/vinyl-fs.d.ts",
-    "src/analyzer.ts",
-    "src/bundle.ts",
-    "src/fork-stream.ts",
-    "src/get-dependencies-from-document.ts",
-    "src/polymer-build.ts",
-    "src/polymer-project.ts",
-    "src/streams.ts",
-    "src/service-worker.ts",
-    "src/path-transformers.ts",
+    "node_modules/typescript/lib/lib.es2016.d.ts",
     "typings/index.d.ts"
   ],
-  "atom": {
-    "rewriteTsconfig": true
-  }
+  "include": [
+    "custom_typings/**/*.ts",
```

Comment:
@rictic do you remember why we use `/// <reference path="../custom_typings/main.d.ts" />` in analyzer.ts rather than include `custom_typings` in tsconfig? Is it for declarations and dependers?